CONTRIBUTING.md

```diff
@@ -112,13 +112,15 @@ Testing Lints
 **Creating Unit Tests.** Every lint should also have corresponding unit
 tests (generally at least one for a success and one for a failure condition). We
 have typically generated test certificates using Go (see
-[documentation][CreateCertificates] for details), but OpenSSL
-could also be used. Test certificates should be placed in `testdata/` and called
-from the test file created by `newLint.sh`. You may want to prepend the PEM with
-the output of `openssl x509 -text`. You can run your lint against a test
-certificate from a unit test using the `test.TestLint` helper function.
+[documentation][CreateCertificates] for details and [example][certGenerator] as
+reference), but OpenSSL could also be used. Test certificates should be placed in
+`testdata/` and called from the test file created by `newLint.sh`. You may want to
+prepend the PEM with the output of `openssl x509 -text`. You can run your lint
+against a test certificate from a unit test using the `test.TestLint` helper
+function.
 
 [CreateCertificates]: https://golang.org/pkg/crypto/x509/#CreateCertificate
+[certGenerator]: ./certGenerator.go
```

Comment:
I'd prefer this doesn't live in the root of the repository. It should probably be under `v3/test`.

Reply:
Also I believe where you've placed it now it's not being built in CI or linted with golangci-lint. We'd definitely want the latter at least, and likely should consider the former to prevent bitrot over time.